.circleci/config.yml

```
@@ -18,7 +18,7 @@ commands:
       - run:
           name: Install Dependencies
           command: |
-            npx lerna bootstrap
+            yarn
             ! git diff --name-only | grep yarn.lock
```

```suggestion
            yarn --frozen-lockfile
```
I added `! git diff --name-only | grep yarn.lock` because lerna didn't have the equivalent of `frozen-lockfile`